gtfs/spec/en/reference.md

```diff
@@ -406,6 +406,7 @@ Methods for occupancy forecasting used to populated `occupancies.txt` are not st
 | `sunday` | Enum| **Required** | Functions in the same way as `occupancies.monday` except applies to Sundays. |
 | `start_date` | Date | **Required** | Start date of the date interval that the occupancy level is valid.<br><br>To define single dates, `start_date` and `end_date` may be the same. |
 | `end_date` | Date | **Required** | End date of the date interval that the occupancy level is valid.<br><br>To define single dates, `start_date` and `end_date` may be the same. |
+| `start_time` | Time | **Conditionally Required** | First stop departure time for a given vehicle on a trip using `frequencies.txt`.<br><br>Must be some multiple (including zero) of `frequencies.headway_secs` plus `frequencies.start_time` for the corresponding time period.<br><br>Conditionally Required:<br>- **Required** for trips using `frequencies.txt`.<br>- **Forbidden** otherwise. |
```

Comment:
This will get very complex. But do we want to do something like start_time end_time so for frequency based trips a producer **could** do occupancy time bands? Self-note to my future self: I just shot myself in the foot.

Reply:
Maybe I am missing something, but isn't `start_time` enough to identify individual "vehicle journeys" on a frequency-based trip, and then define different occupancies per `stop_sequence`, per vehicle journey? This seems to accomplish "occupancy time bands".

Reply:
@scmcca in frequency based operation only exact times will have some useful meaning for start_time.

Reply:
This could be what I am missing. At the risk of sounding misinformed, why would it not work for `exact_times=0`? Can't `frequencies.start_time` be used to estimate vehicle journeys?

Reply:
I think that part is only possible heuristically, hence clamp to the nearest start_time.

Reply:
I agree with @skinkie that specifying a start and end time to define a window is a more natural solution for exact_times=0 trips. 

For exact_times=1, you have specific trips instances that are defined in static GTFS where you can say "A bus is scheduled to arrive in 5 minutes", so start_time unambigously maps to a single trip instance. 

For exact_times=0 trips, static GTFS is just defining an expected frequency (not trip instances at specific times), so (without real-time data) you can only say "Vehicles are expected to arrive every 15 minutes" during this time window. The exact time that these trip instances are expected to occur drifts throughout the day depending on the previous trip instances, so defining just a start_time may get ambiguous and we would need defined heuristics in the spec of how specific cases would be handled.